ci/run-k8s-e2e-tests.sh

```diff
@@ -21,18 +21,22 @@ function echoerr {
 }
 
 RUN_CONFORMANCE=false
+RUN_WHOLE_CONFORMANCE=false
 RUN_NETWORK_POLICY=false
 RUN_E2E_FOCUS=""
 KUBECONFIG_OPTION=""
 E2E_CONFORMANCE_SKIP="\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]|\[sig-cli\]|\[sig-storage\]|\[sig-auth\]|\[sig-api-machinery\]|\[sig-apps\]|\[sig-node\]"
 MODE="report"
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+KUBE_CONFORMANCE_IMAGE_VERSION="$(head -n1 $THIS_DIR/k8s-conformance-image-version)"
```

Comment:
do we need to update the version included in `k8s-conformance-image-version`. We haven't done it in a while and it's still pointing at a beta version.

Reply:
Let's change it to `auto`, which is the default value. I remember we set it to a fixed version because of some bug/issue before.

Reply:
makes sense, thanks